packages/form-builder/addon/gql/mutations/copy-form.graphql

```diff
@@ -0,0 +1,11 @@
+#import FormInfo from '../fragments/form-info.graphql'
+
+mutation CopyForm($input: CopyFormInput!) {
+  copyForm(input: $input) {
+    form {
+      id
+      ...FormInfo
```

Comment:
Does this prevent the extra request after the redirect?

Reply:
Not addressed yet

Reply:
Without the `FormInfo` fragment, we would not receive the `slug` which is necessary as a query param for the redirect. So yes, it's essential here.

Reply:
But does it prevent loading the whole form after the redirect? If not, I'd suggest only fetching the slug (which we'd actually know without the response, just saying) as we only need that and not the whole form data.

**However,** if the prevents the query after we redirect to the new form then I'm all for it as we save ourselves one request :)

Reply:
Makes no difference. I will delete the fragment from the mutation response. Save us 200 *bytes* :tada: